### DISCUSSION TITLE
Loading preprocessor gets stuck using Controlnet

### DISCUSSION THREAD

**Discussion by myrkur12** [TheLastBen/fast-stable-diffusion#1598] [Q&A]
Has anyone got Controlnet (https://github.com/Mikubill/sd-webui-controlnet) to work for them? I installed it via extensions, added a model to the folder, but anytime I try to generate it crashes on this step in the logs:

Loading preprocessor: canny, model: control_sd15_canny [fef5e48e]

Any ideas? Fixes?

**Comment by TheLastBen** [comment]
I'll look into it
